src/DataProbePostProcessing.C

```diff
@@ -758,7 +966,102 @@ DataProbePostProcessing::provide_output_txt(
         }
         else {
           // nothing to do for this probe on this processor
-        } 
+        }
+	} else if (probeInfo->geomType_[inp]==DataProbeGeomType::PLANE) {
+	    // -- Output the plane in text file --
+
+	    // open the file for this probe
+	    const int timeStepCount = realm_.get_time_step_count();
+	    const int processorId = probeInfo->processorId_[inp];
+	    std::ostringstream ss;
+	    ss << std::setw(7)<<std::setfill('0')<<timeStepCount;
+	    ss <<"_"<<processorId;
+	    const std::string fileName = probeInfo->partName_[inp] + "_" + ss.str() + ".dat";
+	    std::ofstream myfile;
+	    if ( processorId == NaluEnv::self().parallel_rank()) {    
+
+	      const int N1 = probeInfo->edge1NumPoints_[inp];
+	      const int N2 = probeInfo->edge2NumPoints_[inp];
+	      const int pointsPerPlane = N1*N2;
+	      const int numPlanes = probeInfo->offsetSpacings_[inp].size();
+
+	      // Get the path to the file name, and create any directories necessary
+	      // - Get the path
+	      size_t pathfound; 
```

Comment:
Please convert lines 990-1007 to C++. `sys/stat.h` is not standard if I remember correctly. Has this been tested on different compilers/operating systems that this will be expected to run on. 

While, this capability would be useful elsewhere too, e.g., SurfaceForceAndMoment, I would prefer to find some portable/standard way of implementing this and have it tested across different platforms. At that point, I would also recommend elevating this to a utility function that we can use in other classes that create text files.